types/node/process.d.ts

```diff
@@ -793,6 +906,12 @@ declare module "process" {
                  * @since v17.3.0, v16.14.0
                  */
                 getActiveResourcesInfo(): string[];
+                /**
+                 * Provides a way to load built-in modules in a globally available function.
+                 * @param id ID of the built-in module being requested.
+                 */
+                getBuiltinModule<ID extends keyof BuiltInModule>(id: ID): BuiltInModule[ID];
+                getBuiltinModule(id: string): object | undefined;
```

Comment:
Something feels weird about it being `object` (I'm always suspicious of that type), but I think it's okay...

Reply:
For me it looks okay. Let's collect the feedback